docs/physics/new_plasma.rst

```diff
@@ -40,9 +41,9 @@ Updating the Plasma
 -------------------
 During each iteration of the main code, TARDIS updates the plasma using the `update_radiationfield` function. This requires, at minimum, new values for `t_rad` (the radiation temperature), `w` (the dilution factor) and `j_blues` (the intensity in the blue part of each line).
 
-
-
+Plasma Calculations
+-------------------
 .. toctree::
     :maxdepth: 2
     
-    plasma/index
+    plasma/index
```

Comment:
I actually meant copy TOC from plasma/index.rst and paste it here, because otherwise linking `plasma/index.rst` will nest its content one more level deep which we don't want - so you just replace line 46-49 with: 
```
.. toctree::
    :maxdepth: 2

    plasma/lte_plasma
    plasma/nebular_plasma
    plasma/macroatom
    plasma/nlte
    plasma/helium_nlte
```
Also please rename `plasma/plasma.rst` to `plasma/old_plasma.rst` and `new_plasma.rst` to `plasma.rst` - and do a sanity check by rebuilding docs. I know this may be bit complicated due to keeping old physics section - feel free to ask if it doesn't turn out well.

Reply:
Done!